pkg/operator/status.go

```diff
@@ -525,83 +501,33 @@ func (optr *Operator) cfeEvalCgroupsV1() (bool, error) {
 	return nodeClusterConfig.Spec.CgroupMode == configv1.CgroupModeV1, nil
 }
 
-// isKubeletSkewSupported checks the version skew of kube-apiserver and node kubelet version.
-// Returns the skew status. version skew > 2 is not supported.
-func (optr *Operator) isKubeletSkewSupported(pools []*mcfgv1.MachineConfigPool) (skewStatus string, coStatus configv1.ClusterOperatorStatusCondition, err error) {
```

Comment:
I pitch dropping this in my commit message, but thoughts about whether I should drop it in the dev branch?  I'd like to, but looking for MCO-maintainer opinions first.

And then, if the MCO-maintainer opinion is "yes, please drop this in dev", I'd like MCO-maintainer opinions on whether you want me to use this same bug-series (because we'll need a dev bug anyway that says "4.19 won't need the bare-RHEL guard, because that's only a 4.18 -> 4.19 issue, so only 4.18's MCO needs that guard"), or if you want it NO-ISSUE, or you want a separate OCPBUGS series?

Reply:
I've opened #4970 to drop this guard in the dev branch, and I'll rebase this pull and close this thread once that's been kicked around, merged, and verified.

Reply:
#4970 merged (thanks!), and I've just pushed 99156806f -> 49511e26f, rebasing this branch on top of the current `release-4.18`, pulling in #4970 changes like dropping kubelet-skew unit tests, and making some orthogonal changes like moving to `node.openshift.io/os_id` label selection for finding the package-managed RHEL nodes.  I'll mark this thread resolved.